src/ck-ldb/LBManager.C

```diff
@@ -764,6 +800,28 @@ void LBManager::pup(PUP::er& p)
   }
 }
 
+extern "C" void configureTreeLB(const char *json_str) {
```

Comment:
Didn't we remove all usage of extern "C" a while back?

Reply:
If it's needed at all if should probably be CMIEXTERNC or whatever our macro for that is

Reply:
Fixed.